src/main/java/uk/gov/hmcts/probate/service/template/pdf/caseextra/decorator/SolicitorCoversheetPDFDecorator.java

```diff
@@ -50,7 +54,16 @@ public String decorate(CaseData caseData) {
                 .build();
             decoration = caseExtraDecorator.combineDecorations(decoration,
                 caseExtraDecorator.decorate(ihtEstate207CaseExtra));
-        }        
+        }
+        if (tcResolutionLodgedWithApplicationRule.isApplicable(caseData)) {
+            TCResolutionLodgedWithAppCaseExtra tcResolutionLodgedWithAppCaseExtra = TCResolutionLodgedWithAppCaseExtra
+                .builder()
+                .showTcResolutionLodgedWithApp(YES)
+                .tcResolutionLodgedWithAppText(TC_RESOLUTION_LODGED_WITH_APP)
+                .build();
+            decoration = caseExtraDecorator.combineDecorations(decoration,
```

Comment:
this combineDecorations call needs to be made on alll the decorations in this class - so they can overlap correctly